draft/2021-02-24-this-week-in-rust.md

```diff
@@ -82,11 +82,13 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 ## Rust Compiler Performance Triage
 
-A mostly quiet week, though with an excellent improvement in bootstrap times, shaving off a couple percent off the total and 10% off of rustc_middle due to changes in the code being compiled.
+Overall, a positive week for compiler performance with only one moderate regression. The change that introduced the regression leads to significantly improved [bootstrap speed](https://github.com/rust-lang/rust/pull/70951#issuecomment-766292996) of the compiler as well as easier maintainability.
 
-Triage done by @simulacrum. Revision range: [ea09825..f1c47c7](https://perf.rust-lang.org/?start=ea098255f74923d69ea234ee526df6b9cecc3b9b&end=f1c47c79fe8438ed241630f885797eebef3a6cab&absolute=false&stat=instructions%3Au)
+Triage done by **@rylev**.
+Revision range: [f1c47c79fe8438ed241630f885797eebef3a6cab..301ad8a4fa3ea56fb980443b7997c8f9d72dd717](https://perf.rust-lang.org/?start=f1c47c79fe8438ed241630f885797eebef3a6cab&end=301ad8a4fa3ea56fb980443b7997c8f9d72dd717&absolute=false&stat=instructions%3Au)
```

Comment:
A line this long will break the formatting when the newsletter is viewed in a mobile browser. I will shorten the revision ids this week, but can you submit with shortened revision ids in the future? (Good example is in this PR https://github.com/rust-lang/this-week-in-rust/pull/1809/files)